cupy/cuda/cupy_cuda.h

```diff
@@ -150,6 +150,8 @@ CUresult cuLaunchKernel(...) {
 
 typedef enum {
     cudaSuccess = 0,
+    cudaErrorInvalidValue = 1,
+    cudaErrorMemoryAllocation = 2,
```

Comment:
Just an additional reference to the direct link https://docs.nvidia.com/cuda/cuda-runtime-api/group__CUDART__TYPES.html#group__CUDART__TYPES_1g3f51e3575c2178246db0a94a430e0038